Tests/survey/test_agent_governance_doctrine_contracts.py

```
@@ -66,15 +83,22 @@ def read_governance() -> str:
 def assert_tracked() -> None:
-    completed = subprocess.run(
-        ["git", "ls-files", "--error-unmatch", "AGENTS.md"],
-        cwd=ROOT,
-        text=True,
-        capture_output=True,
-        check=False,
-    )
-    assert completed.returncode == 0, "AGENTS.md is not tracked by git"
+    for path in (GOVERNANCE, VALIDATOR):
+        relative = path.relative_to(ROOT).as_posix()
+        completed = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)